dev/tests/integration/testsuite/Magento/Framework/App/TestStubs/InterfaceOnlyBackendAction.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\App\TestStubs;
+
+use Magento\Framework\App\ActionInterface;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\View\Result\Page;
+use Magento\Framework\View\Result\PageFactory;
+
+/**
+ * Stub interface action controller implementation for testing purposes.
+ */
+class InterfaceOnlyBackendAction implements ActionInterface
```

Comment:
I think for backend we need to introduce another interface, as before action execution ACL should be additionally checked

Reply:
I'm not sure what do you mean.